guides/common/modules/con_content-views-in-project.adoc

```diff
@@ -19,32 +19,64 @@ However, the repositories for the content views still exist and you can keep man
+endif::[]
+ifdef::almalinux,amazon_linux,centos,oracle_linux,red_hat_enterprise_linux,rocky_linux,suse_linux_enterprise_server[]
+xref:Adding_Custom_RPM_Repositories_{context}[].
+endif::[]
+endif::[]
+endif::[]
+endif::[]
+
 .Promoting a content view across environments
 When you promote a content view from one environment to the next environment in the application lifecycle, {Project} updates the repository and publishes the packages.
 
 .Promoting a package from _Development_ to _Testing_
 ====
-The repositories for _Testing_ and _Production_ contain the `_<my_software>_-1.0-0.{client-pkg-arch}.{client-pkg-ext}` package:
+The repositories for _Testing_ and _Production_ contain the `_my-software_-1.0-0.{client-pkg-arch}.{client-pkg-ext}` package:
 
 |===
 | | Development | Testing | Production
 
 | Version of the content view | Version 2 | Version 1 | Version 1
-| Contents of the content view | _<my_software>_-1.1-0.{client-pkg-arch}.{client-pkg-ext} | _<my_software>_-1.0-0.{client-pkg-arch}.{client-pkg-ext} | _<my_software>_.0-0.{client-pkg-arch}.{client-pkg-ext}
+| Contents of the content view | _my-software_-1.1-0.{client-pkg-arch}.{client-pkg-ext} | _my-software_-1.0-0.{client-pkg-arch}.{client-pkg-ext} | _my-software_1.0-0.{client-pkg-arch}.{client-pkg-ext}
 |===
 
-If you promote Version 2 of the content view from _Development_ to _Testing_, the repository for _Testing_ updates to contain the `_<my_software>_-1.1-0.{client-pkg-arch}.{client-pkg-ext}` package:
+If you promote Version 2 of the content view from _Development_ to _Testing_, the repository for _Testing_ updates to contain the `_my-software_-1.1-0.{client-pkg-arch}.{client-pkg-ext}` package:
 
 |===
 | | Development | Testing | Production
 
-| Version of the content view | Version 2  | *Version 2* | Version 1
-| Contents of the content view | _<my_software>_-1.1-0.{client-pkg-arch}.{client-pkg-ext} | *_<my_software>_-1.1-0.{client-pkg-arch}.{client-pkg-ext}* | _<my_software>_-1.0-0.{client-pkg-arch}.{client-pkg-ext}
+| Version of the content view | Version 2 | *Version 2* | Version 1
+| Contents of the content view | _my-software_-1.1-0.{client-pkg-arch}.{client-pkg-ext} | *_my-software_-1.1-0.{client-pkg-arch}.{client-pkg-ext}* | _my-software_-1.0-0.{client-pkg-arch}.{client-pkg-ext}
 |===
 
 This ensures hosts are designated to a specific environment but receive updates when that environment uses a new version of the content view.
 ====
 
+ifndef::satellite[]
+You can access unprotected repositories in published content view versions.
+The URL consists of your {SmartProxy} FQDN, `/pulp/content/`, your organization label, `/content_views/`, your content view, your content view version, `/custom/`, your product label, `/`, your repository label, and a trailing `/`, for example, `https://{foreman-example-com}/pulp/content/Example/content_views/{client-content-content-view-label}/2.1/custom/{client-content-product-label}/{client-content-repository-label}/`.
```

Comment:
It might be worth noting that this specific functionality can be turned off with a setting 'Distribute archived content view versions'.

Reply:
Added.